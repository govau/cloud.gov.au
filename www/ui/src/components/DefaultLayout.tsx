import * as React from "react";
import styled, { css } from "../styled-components";
import { Flex, Box } from "grid-styled";

import Container from "./Container";

const Wrapper = styled.div`
  min-height: 50vh;
  ${({ theme }) =>
    css`
      background-color: ${theme.contentBgColor};
    `};
`;

interface Props {}

const DefaultLayout: React.StatelessComponent<Props> = ({ children }) => (
  <Wrapper>
    <Container>
      <Flex wrap={true}>
        <Box w={1} pt={3} pb={3}>
          {children}
        </Box>
      </Flex>
    </Container>
  </Wrapper>
);

export default DefaultLayout;
