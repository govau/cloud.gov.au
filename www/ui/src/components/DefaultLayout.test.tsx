import * as React from "react";
import * as renderer from "react-test-renderer";

import DefaultLayout from "./DefaultLayout";

it("renders without crashing", () => {
  const tree = renderer.create(<DefaultLayout>content</DefaultLayout>).toJSON();
  expect(tree).toMatchSnapshot();
});
