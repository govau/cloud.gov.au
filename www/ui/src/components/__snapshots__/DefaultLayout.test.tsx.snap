// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  className="sc-bwzfXH jIcrnc"
>
  <div
    className="sc-bdVaJa liJgVB sc-bdVaJa jipgQh"
  >
    <div
      className="sc-bwzfXH hbZeap"
    >
      <div
        className="sc-bdVaJa gRPNsf"
      >
        content
      </div>
    </div>
  </div>
</div>
`;
